core/base/mtx_io.cpp

```diff
@@ -254,6 +303,11 @@ class mtx_reader {
             std::istream &header, std::istream &content,
             const entry_format *entry_reader,
             const storage_modifier *modifier) const = 0;
```

Comment:
I do not see the storage_modifier defined anywhere. Did I miss that?

Reply:
It was defined in #94, so it doesn't show up on the diff, but you can see it if you open the lines that were not changed: [here](https://github.com/ginkgo-project/ginkgo/pull/98/files#diff-3a5e472edef275112e43e89db004f69cR197).